src/Aspire.Dashboard/Utils/FormatHelpers.cs

```diff
@@ -10,36 +10,36 @@ namespace Aspire.Dashboard.Utils;
 
 internal static partial class FormatHelpers
 {
+    // There are an unbound number of CultureInfo instances so we don't want to use it as the key.
+    // Someone could have also customized their culture so we don't want to use the name as the key.
+    // This struct contains required information from the culture that is used in cached format strings.
+    private record struct CultureDetailsKey(string LongTimePattern, string ShortDatePattern, string NumberDecimalSeparator);
```

Comment:
Looks like this can be `readonly`, which simplifies the generated code a little.

```suggestion
    private readonly record struct CultureDetailsKey(string LongTimePattern, string ShortDatePattern, string NumberDecimalSeparator);
```